ux.symfony.com/templates/ux_packages/toggle_password.html.twig

```diff
@@ -0,0 +1,38 @@
+{% extends 'packageBase.html.twig' %}
+
+{% block component_header %}
+    {% component PackageHeader with {
+        package: 'toggle-password',
+        eyebrowText: 'Switch between password & text'
+    } %}
+        {% block title_header %}
+            From password to text
+            <span class="playfair ps-2">& vice versa</span>
+        {% endblock %}
+
+        {% block sub_content %}
+            Upgrade your password field to toggable field between text and password.
+        {% endblock %}
+    {% endcomponent %}
+{% endblock %}
+
+{% block code_block_left %}
+    <twig:CodeBlock filename="src/Form/TogglePasswordForm.php" />
+{% endblock %}
+
+{% block code_block_right %}
+    <twig:CodeBlock
+            filename="templates/ux_packages/toggle_password.html.twig"
+            targetTwigBlock="demo_content"
+    />
+{% endblock %}
+
+{% block demo_title %}UX Toggle Password{% endblock %}
+
+{% block demo_content %}
+    {{ form_start(form) }}
+    {{ form_row(form.email) }}
+    {{ form_row(form.password) }}
+    <button type="submit" class="btn btn-primary">Login</button>
```

Comment:
I usually indent the `form_row` and `button` - as if `form_start()` was actually a `<form>` tag. Subjective though

Reply:
I had a discussion with @HeahDude about this (he shares your opinion) and i agree with both of you, i like the logic 👍🏼